### ISSUE TITLE
aria-hidden="true" doesn’t work

### ISSUE THREAD

**Issue by WindyHard** [readest/readest#6206]
### Bug Description

aria-hidden="true" doesn’t work — the attribute has no effect; the screen reader still reads it out.

### Steps to Reproduce

<p aria-hidden="true">This should not be read out; it is purely decorative.</p>

### Expected Behavior

This text should be visible to sighted users, but screen readers should ignore it.

### Operating System

Android

### OS Version

android 14

### Readest Version

version0.12.8 (webview 151.0.7922.199)

### Custom CSS

* {
  margin: 0;
  padding: 0;
  box-sizing: border-box;
}
p {
    line-height: 1.8;
    font-size: 1em;
	text-indent: 2em;
    margin: .8em 0 0 0;
}

**Comment by chrox** [comment]
Please specify the component and provide more information without which we cannot verify and fix this.
